standup/status/jinja2/base.html

```diff
@@ -72,9 +73,10 @@
                     {% if user.is_staff -%}
                       <li><a href="{{ url('admin:index') }}">Admin</a></li>
                     {%- endif %}
+                    <li><a href="{{ url('users.logout') }}">Logout</a></li>
```

Comment:
Previously, there was no way to log out, so I added that. This really should be a POST action with a csrf, though.